### DISCUSSION TITLE
Validation Error when running examples

### DISCUSSION THREAD

**Discussion by lardratboy** [gfx-rs/wgpu#2387] [Q&A] [answered]
I have been seeing validation errors when starting the wgpu examples. Some of the examples continue to run and bring up a window and show expected content for the selected demo, but others like bunnymark bring up the window and no bunnies...  oops I forgot the question, anyone also have their monitors in portrait mode and does this happen for you?

running : 

cargo run --example boids

produces the following output:

    Finished dev [unoptimized + debuginfo] target(s) in 0.18s
     Running `target\debug\examples\boids.exe`
Using NVIDIA GeForce GTX 1080 Ti (Vulkan)
[2022-01-12T21:32:57Z ERROR wgpu_hal::vulkan::instance] VALIDATION [VUID-VkSwapchainCreateInfoKHR-imageExtent-01274 (0x7cd0911d)]
        Validation Error: [ VUID-VkSwapchainCreateInfoKHR-imageExtent-01274 ] Object 0: handle = 0x2ae58c287b0, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x7cd0911d | vkCreateSwapchainKHR() called with imageExtent = (1904,990), which is outside the bounds returned by vkGetPhysicalDeviceSurfaceCapabilitiesKHR(): currentExtent = (800,600), minImageExtent = (800,600), maxImageExtent = (800,600). The Vulkan spec states: imageExtent must be between minImageExtent and maxImageExtent, inclusive, where minImageExtent and maxImageExtent are members of the VkSurfaceCapabilitiesKHR structure returned by vkGetPhysicalDeviceSurfaceCapabilitiesKHR for the surface (https://vulkan.lunarg.com/doc/view/1.2.198.1/windows/1.2-extensions/vkspec.html#VUID-VkSwapchainCreateInfoKHR-imageExtent-01274)
[2022-01-12T21:32:57Z ERROR wgpu_hal::vulkan::instance]         objects: (type: DEVICE, hndl: 0x2ae58c287b0, name: ?)

**Comment by aloucks** [comment, marked as answer]
There's a bug in winit. 

- https://github.com/rust-windowing/winit/issues/2094
- https://github.com/gfx-rs/wgpu/issues/2286
